### ISSUE TITLE
loss function diff

### ISSUE THREAD

**Issue by WangliLin** [lyakaap/Kaggle-Carvana-3rd-Place-Solution#8]
could you please tell me why the implement of loss function [weight_bce_dice_loss] is different from your kernel in kaggle : [kernel](https://www.kaggle.com/lyakaap/weighing-boundary-pixels-loss-script-by-keras2)

**diff 1: weighted bce loss**
[kaggle kernel]
`loss = (1. - y_true) * logit_y_pred + (1. + (weight - 1.) * y_true) * \
    (K.log(1. + K.exp(-K.abs(logit_y_pred))) + K.maximum(-logit_y_pred, 0.))`
[github repo]
`loss = weight * (logit_y_pred * (1. - y_true) + 
                     K.log(1. + K.exp(-K.abs(logit_y_pred))) + K.maximum(-logit_y_pred, 0.))
`

**diff2: weightd bce dice loss**
[kaggle kernel]
`border = K.cast(K.greater(averaged_mask, 0.005), 'float32') * K.cast(K.less(averaged_mask, 0.995), 'float32')`
[github repo]
`weight = 5. * K.exp(-5. * K.abs(averaged_mask - 0.5))`
